allensdk/brain_observatory/nwb/nwb_utils.py

```diff
@@ -0,0 +1,50 @@
+# All of the omitted stimuli have a duration of 250ms as defined
+# by the Visual Behavior team. For questions about duration contact that
+# team.
+omitted_stimuli_duration = 0.250
```

Comment:
This should probably be a default argument to `set_omitted_stop_time`. e.g.

```
def set_omitted_stop_time(stimulus_table_row: dict, omitted_stimuli_duration: float = 0.250):
```

Reply:
Set it to default and removed module global.